dev/io.openliberty.microprofile.telemetry.internal.common/src/io/openliberty/microprofile/telemetry/internal/interfaces/OpenTelemetryAccessor.java

```diff
@@ -42,11 +42,10 @@ public class OpenTelemetryAccessor {
      *         is disabled or the application has shut down.
      */
     public static OpenTelemetryInfoInternal getOpenTelemetryInfo() {
-        //TOOD when the SPI is out of beta, make this a redirect to the SPI accessor to avoid code duplication
```

Comment:
I did this inverted, as most calls will come through this class so redirecting into it saves one more hop for the most used path.